data/scrape/lib/generate_tsv_summary.py

```diff
@@ -15,8 +15,8 @@
 )
 
 
-def _wiki_name_and_transcription_level(ele: List[str]) -> str:
-    return ele[3] + ele[7]
+def wiki_name_filtered_and_path(ele: List[str]) -> str:
```

Comment:
Would it not suffice to just sort by the path? That'd make this function unnecessary, could do `operator.itemgetter(0)` instead, which is way faster too.

Reply:
Perhaps, but then we'd be sorting by ISO code and get things like Irish ending up next to German.